front/lib/api/assistant/conversation.ts

```diff
@@ -210,7 +209,7 @@ export async function deleteConversation(
   if (destroy) {
     await conversation.delete(auth);
   } else {
-    await conversation.updateVisiblity("deleted");
+    await conversation.markAsDeleted();
```

Comment:
NIT: I would call it conversation.updateVisibilityDeleted() so that we know exactly what it does, feel free to ignore!